lib/node_modules/@stdlib/stats/base/cumax/examples/index.js

```diff
@@ -18,22 +18,16 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
 var Float64Array = require( '@stdlib/array/float64' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
 var cumax = require( './../lib' );
 
-var y;
-var x;
-var i;
-
-x = new Float64Array( 10 );
-y = new Float64Array( x.length );
-for ( i = 0; i < x.length; i++ ) {
-	x[ i ] = round( randu()*100.0 );
-}
+var x = discreteUniform( 10, 0, 100, {
+	'dtype': 'float64'
+});
+var y = new Float64Array( x.length );
 console.log( x );
 console.log( y );
 
-cumax( x.length, x, 1, y, -1 );
+cumax( x.length, x, 1, y, 1 );
```

Comment:
```suggestion
cumax( x.length, x, 1, y, -1 );
```
Ensure that README and this example match.